rasa/core/brokers/pika.py

```diff
@@ -337,7 +337,10 @@ def _message_properties(self) -> "BasicProperties":
 
         kwargs = {"app_id": self.rasa_environment} if self.rasa_environment else {}
 
-        return BasicProperties(**kwargs)
+        return BasicProperties(
+            delivery_mode = 2, # make message persistent
+            **kwargs
+        )
```

Comment:
I think `black` might still fail for this wrapping (unless the command `# make persistent...` were longer)